README.md

```diff
@@ -64,10 +65,9 @@ For testing the running time as functions of n and m, you should also write code
 
 ## Part 2: mappers
 
-Now write the tools for exact pattern matching. You can use the naive algorithm to test your linear time algorithm; the result of the two programs that you write should be identical after you sort the output.
+Now write the tools for exact pattern matching. You can use the naive algorithm to test your linear time algorithm; the result of the two programs that you write should be identical after you sort the output.> ./naive fasta.fa fastq.fq | sort > naive.sam
```

Comment:
Why did you put a command line at the end of a line in a paragraph? That is a weird thing to do...